projects/tuist/fixtures/app_with_spm_dependencies/App/Project.swift

```diff
@@ -70,4 +79,22 @@ let project = Project(
         ),
     ],
     schemes: Scheme.allSchemes(for: ["App", "AppKit"], executable: "App")
+    /* +
+     [
```

Comment:
Still need to work on scheme part of the feature which will allow you to create a scheme with external targets. This work can be done in an independent PR as well